### DISCUSSION TITLE
Audi Q2 MIB2 MHI2_ER_AU276_P5604

### DISCUSSION THREAD

**Discussion by Vengo222** [Mr-MIBonk/M.I.B._More-Incredible-Bash#501] [General]
Is it possible to create a patch for this software version, with the backup files created by the M.I.B program? , And in that case what would be the procedure or can I send you the backup copy and you create the patch.

firmware: MHI2_ER_AU276_P5604

I can also update to a previous version, but I would have to get some of the files that I indicate, does anyone have the files?

MHI2_ER_AU276_P5515_MU1488
MHI2_ER_AU276_P5510_MU1477
MHI2_ER_AU276_P5509_MU1474
MHI2_ER_AU276_P5600_MU1498

Thanks

**Comment by pssnyman** [comment]
Hi @Vengo222 

Do you happen to have a copy of the original MHI2_ER_AU276_P5600_MU1498 firmware?

Happy to share my MHI2_ER_AU276_P5600_MU1498 backup files if you can help me with the original firmware please?

**Reply by mmmrvn** [reply]
Hi @pssnyman, could you send me MHI2_ER_AU276_P5604 as well if you were able to obtain it? mibsolution.one still down, thanks in advance!

**Comment by Tommick70** [comment]
Can you send me firmware: MHI2_ER_AU276_P5604?

**Reply by mmmrvn** [reply]
Hi @Tommick70, could you send me MHI2_ER_AU276_P5604 as well if you were able to obtain it? mibsolution.one still down, thanks in advance!

**Comment by sushant-in** [comment]
Hey folks, 
If anyone has firmware for MHI2_ER_AU276_P5604 please share.
